packages/datadog-instrumentations/src/cookie-parser.js

```diff
@@ -10,7 +10,7 @@ function publishRequestCookieAndNext (req, res, next) {
     if (cookieParserReadCh.hasSubscribers && req) {
       const abortController = new AbortController()
 
-      const mergedCookies = Object.assign({}, req.cookies, req.signedCookies)
+      const mergedCookies = { ...req.cookies, ...req.signedCookies }
```

Comment:
is that equivalent performance wise ?

Reply:
It's faster

Reply:
Like with every new language feature, these things tend to shift over time as the JS engine matures